blog/dev_hackathon_v4_release.md

```diff
@@ -0,0 +1,134 @@
+(sprint_testimonial)=
+# Dev Sprint: Getting the PyMC v4.0 Release Out
+
+:::{post} Jun 24, 2022
+:tags: sprint, contributing
+:category: news
+:author: Reshama Shaikh, Thomas Wiecki
+:image: 0
+:exclude:
+
+The PyMC team coordinated a development hackathon in May/June 2022 to complete updates and documentation in order to release [v4.0 of the PyMC library](https://twitter.com/pymc_devs/status/1533863780601446401).
+:::
+
+![v4 release social media card](../_static/hackathon_v4_release/v4release_card.png)
+
+The [v4.0 release of the PyMC library](https://twitter.com/pymc_devs/status/1533863780601446401) was officially announced on June 6, 2022.  To achieve this milestone, the PyMC team planned a week-long online sprint beginning May 30, 2022.  The goal was to complete remaining issues on the v4.0 release.  But, *mainly* it was to finalize the documentation and update the website for PyMC.
+
+There are 3 separate websites for [PyMC](https://www.pymc.io/welcome.html):  
+- [Website and Blog](https://www.pymc.io/welcome.html)
+- [Documentation (API)](https://www.pymc.io/projects/docs/en/stable/api.html)
+- [Examples](https://www.pymc.io/projects/examples/en/latest/gallery.html)
+
+And all 3 websites had separate navigation bars and it was confusing for the users, maybe for the library maintainers too?
+
+This Dev sprint was held on the new PyMC Discord server which easily enables sharing video, camera and screen share and provides an easy way to hop between “channels”.  
+
+We shared the sprint events on our new [PyMC Online Meetup](https://www.meetup.com/pymc-online-meetup/events/286202420/) to involve experienced contributors in the community as well. We had a mix of core contributors and newcomers, which is a great combination, as it enables us to be productive, but also to foster the new generation of core developers. The following team members attended, from various locations around the world:  
+
+- Thomas Wiecki (Germany)
+- Ricardo Vieira (Romania)
+- Christian Luhmann (New York, USA)
+- Alex Andorra (France)
+- Michael Osthege (Germany)
+- Oriol Abril Pla (Finland)
+- Chris Fonnesbeck (Nashville, USA)
+- Osvaldo Martin (Argentina)
+- Rob Zinkov (Germany)
+- Reshama Shaikh (New York, USA)
```

Comment:
At the very least Max @ferrine is missing from the list. I only attended part of a couple of the sessions so there might be more people missing.